.gitignore

```diff
@@ -1,10 +1,74 @@
-*.iml
-.gradle
-/local.properties
-/.idea
 .DS_Store
+
+.gradle
+**/build/
```

Comment:
This is also redundant with `build/`. We can safely remove this one.

You can check [this link](https://stackoverflow.com/a/59789360) for a detailed clarification.

Reply:
are u sure about that one, because **/build will match to files and folders that are **not** in the root of the project(where the gitignore is) unlike build/

Reply:
as agreed removed it ✅ aa `build/` matches throughout the project